api_app/script_analyzers/general.py

```diff
@@ -104,14 +108,26 @@ def start_analyzers(
                 ]
             # run analyzer with a celery task asynchronously
             stl = ac.get("soft_time_limit", 300)
-            send_task(
-                "run_analyzer",
-                args=args,
-                queue=queue,
-                soft_time_limit=stl,
+            t_id = uuid()
+            tasks.run_analyzer.apply_async(
```

Comment:
Is there a particular reason why you changed this? The reason for using `send_task` was to avoid cyclic import of `tasks` module. Is it not possible to send `task_id` when using `send_task`?

Reply:
`apply_async` provides an argument to set task id, all examples I saw were using it. 
But it actually calls `send_task` under the hood. So yes, it should be possible to continue using it.
https://github.com/celery/celery/blob/a5357cab8aa80ff701a1970f55dc1e1083a161f5/celery/app/task.py#L561-L566

Also, I'll change it to import from the `app` instead of `execute` module, it works on my machine.
Ref: https://docs.celeryproject.org/en/v4.4.7/internals/deprecation.html#modules-to-remove